Solutions/Intel471/Playbooks/Intel471-ImportMalwareIntelligenceToSentinel/README.md

```diff
@@ -0,0 +1,151 @@
+
+[azuredeploy.json](azuredeploy.json) Azure Resource Manager template (ARM template) is responsible
+for building the connector's Logic App along with auxiliary resources. The ARM builds following components:
+
+- **[Logic App](https://docs.microsoft.com/en-us/azure/sentinel/create-custom-connector#connect-with-logic-apps)** responsible for fetching the data from Titan API and ingesting them into `ThreatIntelligenceIndicator` table
+- **[Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/basic-concepts)** responsible for securely storing Titan API credentials
+- **[Blob storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)** responsible for persisting data such as cursor between the API calls
+- Connection objects
+  - Logic app to Key Vault *(authorized automatically)*
+  - Logic app to Blob storage *(authorized automatically)*
+  - Logic app to Microsoft Security Graph *(**requires manual configuration**)*
+
+![Intel 471 Malware Intelligence Logic App](malware-intelligence-screenshot.png "Intel 471 Malware Intelligence Logic App")
+
+## Prerequisites
+
+1. An active account in Titan platform, which is available as part of Intel 471's subscriptions. For more information, please contact sales@intel471.com.
+2. Titan API credentials.
+
+## Deployment instructions
+
+1. To deploy the Playbook, click the **Deploy to Azure** button. It will launch the ARM Template deployment wizard.
+2. Provide following parameters:
+    * **Playbook Name**: Either leave the default one or change it as needed
+    * **Titan User Name**: Titan API username
+    * **Titan API Key**: Titan API key
+    * **Look Back Days**: How many days of history should be pulled on the first run. Leave 0 to start from the current time
+
+    [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FIntel471%2FPlaybooks%2FIntel471-ImportMalwareIntelligenceToSentinel%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FIntel471%2FPlaybooks%2FIntel471-ImportMalwareIntelligenceToSentinel%2Fazuredeploy.json)
+
+## Post-deployment instructions
+
+1. Go to created `Logic App → Edit`. Navigate to `Collect Indicators → Submit multiple tiIndicators` block. Inside this block create a new connection. Select `Connect with managed identity`, provide connection name and click `Create`.
+2. Navigate to `Collect Indicators → BuildTiIndicatorsPayload → Append to array variable` block. Provide desired `action` for incoming indicators. You can choose from `allow`, `alert`, `block` and `unknown`.
+3. Optionally, in the same block, change `targetProduct`. By default, it's `Azure Sentinel`, but it can be changed to `Microsoft Defender ATP`.
```

Comment:
@mmolenda, please update the text from Azure Sentinel to Microsoft Sentinel, thanks.

Reply:
According to [Microsoft Graph Security docs](https://learn.microsoft.com/en-us/connectors/microsoftgraphsecurity/) the API expects `targetProduct` to be either `Azure Sentinel` or `Microsoft Defender ATP`, not `Microsoft Sentinel`. I also tried to set it as `Microsoft Sentinel`, but then I get following error:

```
{
  "error": {
    "code": "",
    "message": "Value cannot be null.\r\nParameter name: key",
    "innerError": {
      "date": "2023-01-26T13:34:34",
      "request-id": "bac89246-b33f-469b-983b-0b36e0a3e8a6",
      "client-request-id": "bac89246-b33f-469b-983b-0b36e0a3e8a6"
    }
  }
}
```

Reply:
@mmolenda, we will check if we can add exception for azure sentinel text fir this.